tools/go.mod

```diff
@@ -1,9 +1,11 @@
 module tools
 
-go 1.21
+go 1.22.1
+
+toolchain go1.23.0
```

Comment:
this is an odd change

Reply:
Ooops, I let a little too much of the automated changes get committed.